index.Rmd

```diff
@@ -19,11 +19,33 @@ This WDL workflow development guide shows a bioinformatics workflow developer ho
 
 ## Target Audience
 
-The course is intended for first time developers of the [WDL workflow language](https://github.com/openwdl/wdl), who wants to iteratively develop a WDL bioinformatics workflow. The audience should be able to comprehend WDL syntax, and should be able to run a WDL workflow on a computing engine of their choice, such as Cromwell, miniWDL, or a cloud computing environment such as Terra, AnVIL, or Dockstore. 
+The course is intended for first time developers of the [WDL workflow language](https://github.com/openwdl/wdl), who wants to iteratively develop a WDL bioinformatics workflow. In order to use this guide the audience should be able to comprehend introductory WDL syntax, and should be able to run a WDL workflow on a computing engine of their choice, such as Cromwell, miniWDL, or a cloud computing environment such as Terra, AnVIL, or Dockstore. 
+
+## Why WDL?
+
+You may have encountered other workflow tools, such as make and snakemake, and those are highly capable. Why learn a brand new workflow language? Let's review some WDL Pros and Cons:
```

Comment:
`make` usually refers to [GNU make](https://www.gnu.org/software/make/), which isn't really a workflow tool. I'd recommend just referencing snakemake, or adding in CWL or Nextflow perhaps.